packages/react-dom-bindings/src/server/ReactDOMServerFormatConfig.js

```diff
@@ -156,6 +158,7 @@ export function createResponseState(
   bootstrapScriptContent: string | void,
   bootstrapScripts: $ReadOnlyArray<string | BootstrapScriptDescriptor> | void,
   bootstrapModules: $ReadOnlyArray<string | BootstrapScriptDescriptor> | void,
+  externalRuntimeConfig: string | BootstrapScriptDescriptor | void,
```

Comment:
Style / naming - would it make sense to name something this like `staticRuntimeConfig : StaticRuntimeConfig` and have an explicit src? Not familiar with this codebase's style, so please ignore if this contradicts
```
type StaticRuntimeConfig = {
  src: string | BootstrapScriptDescriptor,
}
```

Reply:
Not sure, I'll defer to @gnoff and @sebmarkbage on the API shape. My best guess was to copy the convention for the other options, since it makes sense to me that they would be consistent.

Reply:
the `BootstrapScriptDescriptor` isn't a src type so I wouldn't factor it that way.

I would probably just rename it to `externalRuntimScript` and leave the type as is. if it is a string it is implicitly src. if you pass an object it needs to have a src.

Reply:
In general, we prefer few objects for perf.